static-nginx/Dockerfile

```diff
@@ -18,7 +16,8 @@ RUN apk --no-cache --update --repository http://dl-3.alpinelinux.org/alpine/edge
 && rm -rf /var/cache/apk/* /tmp/*
 
 # Set up user
-RUN addgroup -S www-data \
+RUN (delgroup www-data || true) && \
```

Comment:
Found this fix [here](https://github.com/laradock/laradock/issues/2160#issuecomment-909637173)